wiki/Performance_Points/en.md

```diff
@@ -87,7 +87,7 @@ This means that only your highest pp score will award the full amount of pp for
 
 To provide an example of this with a mathematical formula:
 
-Let _PP_ represent each score's individual pp value. _PP\[i\]_ denotes the _i_th score's pp value, sorted descending, where _i_ ranges from 1 to _n_, and _n_ is defined as the amount of scores you have.
+Let *PP* represent each score's individual pp value. *PP\[i\]* denotes the *i*th score's pp value, sorted descending, where *i* ranges from 1 to *n*, and *n* is defined as the amount of scores you have.
```

Comment:
What's the point of this? They're equally valid Markdown.

Reply:
> What's the point of this? They're equally valid Markdown.
![image](https://user-images.githubusercontent.com/46295491/50574353-ed552280-0e21-11e9-8bd5-0301f65aaad8.png)

Reply:
This is the fault of the Markdown engine, and not the syntax itself, I can confirm from GitHub's Markdown engine the following syntax renders fine.

Reply:
Underscore parsing requires space before and after ```i```. For many Markdown, not just osu-web.
e.g. _i_and*i*   ```_i_and*i*```
I should not change all underscore to asterisk. The only problem is on ```i```.

Reply:
@TsukiPoi that's not true, it works with either for GitHub
![screenshot from 2019-01-05 01-47-55](https://user-images.githubusercontent.com/14976516/50702479-06f9a280-108c-11e9-8b69-1e48d4dc8239.png)
![screenshot from 2019-01-05 01-48-03](https://user-images.githubusercontent.com/14976516/50702513-17aa1880-108c-11e9-8d4a-ef35e91a8c04.png)

Reply:
You have to use asterisks with italicizing part of a term. Italicizing a whole term works fine.
(e.g. `Markdownis_hard_` -> Markdownis_hard_ | `Markdownis*hard*` -> Markdownis*hard*)

https://www.markdownguide.org/basic-syntax/#italic